### DISCUSSION TITLE
Nested asset groups

### DISCUSSION THREAD

**Discussion by gnilrets** [dagster-io/dagster#24083] [Ideas]
I would really like support for nesting asset groups.  For example, I'd like to put all of my dlt pipelines in one asset group, with nested groups for the pipeline application (e.g., salesforce, netsuite, etc.).  I think this could be accomplished by simply supporting group names that a `/` in them.  For example

```
@asset(group_name='dlt/salesforce')
def dlt_salesforce_asset():
    ...

@asset(group_name='dlt/netsuite')
def dlt_netsuite_asset():
    ...
```

Then, in the dagster UI, we'd see just one `dlt` group by default, which we could then click on to expand the two subgroups.

I know this can kind of be accomplished using code locations, but setting up a new code location comes with a lot of extra overhead.  Having nestable asset groups would really help with organizing the UI.

**Comment by garethbrickman** [comment]
Feature request is here: https://github.com/dagster-io/dagster/issues/14530
